assets/javascripts/discourse/templates/components/remind-assigns-frequency.hbs

```diff
@@ -6,7 +6,7 @@
       @valueProperty="value"
       @content={{availableFrequencies}}
       @value={{selectedFrequency}}
-      @onChange={{action (mut user.custom_fields.remind_assigns_frequency)}}
+      @onChange={{action "updateSelectedFrequency"}}
```

Comment:
We're gonna codemod away the `action` helper at some point, but if we're already changing this line…

```suggestion
      @onChange={{this.updateSelectedFrequency}}
```